### PR TITLE
LibRetro.RetroArch.Nightly version Nightly

### PR DESCRIPTION
**Pull request by fpscan**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59080)

### CODE REVIEW

**Comment by fpscan** [comment]
I'm thinking it stops because we have unsigned installer(?) I have lowest UAC on my regular environment and it works fine:
image

We couldn't do any signed installer because no one knows how to do it. Is there anyway to fix this in someway around in winget?

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+Installers:
+- Architecture: x64
+  InstallerType: exe
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml:8]:
```suggestion
  InstallerType: nullsoft
```

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+PackageVersion: Nightly
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://buildbot.libretro.com/nightly/windows/x86_64/RetroArch-Win64-setup.exe
+  InstallerSha256: 3F82C92F3450168D3D6FDDEDBCD4810C9EAF1462179E8B0A5A9963C71F47257E
+  InstallerSwitches:
+    Silent: /S
+- Architecture: x86
+  InstallerType: exe
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml:14]:
```suggestion
  InstallerType: nullsoft
```

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://buildbot.libretro.com/nightly/windows/x86_64/RetroArch-Win64-setup.exe
+  InstallerSha256: 3F82C92F3450168D3D6FDDEDBCD4810C9EAF1462179E8B0A5A9963C71F47257E
+  InstallerSwitches:
+    Silent: /S
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml:12]:
```suggestion
```

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+PackageLocale: en-US
+Publisher: LibRetro
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml:7]:
```suggestion
Publisher: Libretro
```
This must match Control Panel.

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.10.3.0
```
This must match Control Panel.

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.yaml:5]:
```suggestion
PackageVersion: 1.10.3.0
```
This must match Control Panel.

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml:5]:
```suggestion
PackageVersion: 1.10.3.0
```
This must match Control Panel.

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  InstallerUrl: https://buildbot.libretro.com/nightly/windows/x86_64/RetroArch-Win64-setup.exe
+  InstallerSha256: 3F82C92F3450168D3D6FDDEDBCD4810C9EAF1462179E8B0A5A9963C71F47257E
+  InstallerSwitches:
+    Silent: /S
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://buildbot.libretro.com/nightly/windows/x86/RetroArch-Win32-setup.exe
+  InstallerSha256: 6F835027B26BD044F0065A9EA65DF3838BC651E73C48F4946078E6B739C77D15
+  InstallerSwitches:
+    Silent: /S
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.installer.yaml:18]:
```suggestion
```

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+PackageLocale: en-US
+Publisher: LibRetro
+PackageName: RetroArch
+License: GPL 3.0
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml:9]:
```suggestion
License: GPL-3.0
```

File: manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml [hunk 10]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+PackageLocale: en-US
+Publisher: LibRetro
+PackageName: RetroArch
```
Comment (ItzLevvie) [inline manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml:8]:
```suggestion
PackageName: RetroArch Nightly
```

Comment (ItzLevvie) [reply manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml:8]: putting this here so people on production version doesn't get automatically upgraded to the nightly version

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
> I'm starting to believe it's not a signing issue rather than Validation-SmartScreen issue. So is it possible to request a review from an engineer?

Logs say this
```
2022-04-29 07:55:44.016 [CLI ] Package hash verification failed. SHA256 in manifest [3f82c92f3450168d3d6fddedbcd4810c9eaf1462179e8b0a5a9963c71f47257e] does not match download [74626acb3d237eaca5aa27ca0fb12325cf7d1d232fa41aa2fb94dde466dc21f0]
```

It looks like Azure Pipelines is downloading a corrupted file but it does install fine on a VM.

**Pull request closed without merging**